src/lesson9/javaBeginnerChapter2/task28/Box.java

```diff
@@ -0,0 +1,17 @@
+package lesson9.javaBeginnerChapter2.task28;
+
+public class Box<T> {
+    private T[] array;
```

Comment:
Your example works, but main idea of the task was to show that we can use T type and not specify some type. As result we can put array of strings in one implementation of Box class and array of intergers in another one.

Reply:
fixed